build.cake

```diff
@@ -145,7 +145,9 @@ Task("Build")
             NoRestore = true,
             MSBuildSettings = new DotNetMSBuildSettings
             {
-                Version = packageVersion
+                Version = packageVersion,
+                FileVersion	= packageVersion,
```

Comment:
```suggestion
                FileVersion = packageVersion,
```

It doesn't show in github, but there is a TAB between `FileVersion` and the `=` token.